src/elements/OCDrop.vue

```diff
@@ -0,0 +1,117 @@
+<template>
+  <div :uk-drop="$_ocDrop_props">
+    <div v-if="$slots.default" class="uk-card uk-card-default uk-card-small uk-card-body">
+      <slot />
+    </div>
+    <slot v-else name="special" />
+  </div>
+</template>
+<script>
+/**
+ * Position any element in relation to another element.
+ */
+export default {
+  name: "oc-drop",
+  status: "review",
+  release: "1.0.0",
+  props: {
+    /**
+     * CSS selector of the element to maintain the drop's visibility.
```

Comment:
I don't understand what this is for. Needs better explanation or maybe even an example. THX

Reply:
take care off